src/RiotAPILibraries.ts

```diff
@@ -68,7 +69,8 @@ export default class RiotAPILibraries {
                 "Content-Type": "application/json",
             },
         };
-        this.allTagOptions = Array.from(this.settings.riotApiLibraries.requiredTagContextMap.values()).flat();
+        this.allTagOptions =([] as string[]).concat(...this.settings.riotApiLibraries.requiredTagContextMap.values());
```

Comment:
I get: `this.settings.riotApiLibraries.requiredTagContextMap.values is not a function`
I ended up changing this to Object.values(this.settings.riotApiLibraries.requiredTagContextMap) to get a little further.

Reply:
right ok this is because the map isn't constructed properly from the json shared_settings. I need to fix the json object to map conversion; or leave it as an object but I'm not sure what the typing looks like on that